package.json

```diff
@@ -106,6 +106,7 @@
     "phone": "git+https://github.com/Automattic/node-phone.git#1.0.8",
     "photon": "2.0.0",
     "postcss-cli": "2.5.1",
+    "postcss-custom-properties": "6.1.0",
```

Comment:
I think we need to regen shrinkwrap here? `npm run update-deps`

Reply:
OK, I reverted back to a state prior to this PR and ran `npm install --save` for `postcss-custom-properties` again. This time it triggered the shrinkwrap update automagically like it should as per the docs. Not sure why that didn't happen the first time around, wondering if it has to do with me editing the css build steps in the package.json at the same time.

Either way this time it only added the dependencies it actually should. So I decided to do an interactive rebase, delete the old shrinkwrap commit and add the new one. Issue should be resolved now 👍

Reply:
Awesome work - this looks much more reasonable, thanks for looking in to this! :)